hyperspy/tests/io/test_hdf5.py

```diff
@@ -289,6 +289,25 @@ def tearDown(self):
         remove(self.filename)
 
 
+class TestAxesConfiguration:
+
+    def setUp(self):
+        self.filename = 'testfile.hdf5'
+        s = BaseSignal(np.zeros((2, 2, 2, 2, 2)))
+        s.axes_manager.signal_axes[0].navigate = True
+        s.axes_manager.signal_axes[0].navigate = True
```

Comment:
Do I understand correctly that now, creating a `BaseSignal` with a numpy array and no additional information will default all axes to `signal_axes` instead of only the last one, like it was before?
Because for me these three lines do not work, because the last line has no signal axis to convert anymore.